pkg/js/helpers.js

```diff
@@ -156,6 +170,18 @@ function ALIAS(name, target) {
     }
 }
 
+// CAA(name,tag,value, recordModifiers...)
+function CAA(name, tag, value){
+    checkArgs([_.isString, _.isString, _.isString], arguments, "CAA expects (name, tag, value)")
```

Comment:
- The flag field is actually a bitfield of 8 bits. Right now only the first bit is used for critical, but in the future there might be multiple flags active at once. Since we already have variable meta options, making flags variable as well could be hard to implement.
  - One possible way is to define CAA_CRITICAL as an integer instead of a meta setter function, and add a special case to `addRecord` similar to how MX priority is implemented right now. The user then can use binary operator to combine flags (`CAA_CRITICAL | CAA_FUTURE_USE`)
- The above suggestion also help users in translating to dnscontrol format as they also can write the numbers directly, but they would need to add it at the end instead of the given position. The docs could indicate this by writing the method definition as `CAA(name, tag, value[, flags], recordModifiers...)`.